### PR TITLE
[workers] add migration guide for v3

### PR DESCRIPTION
**Pull request by admah**
Adds instructions to help users update v2 to v3.

### CODE REVIEW

File: content/workers/wrangler/migration/update-v2-to-v3.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Migrate from Wrangler v2 to v3
+pcx_content_type: how-to
+weight: 1
+---
+
+# Migrate from Wrangler v2 to v3
+
+There are no special instructions for migrating Wrangler v2 to v3. You should be able to update the package via [our update guide](/workers/wrangler/install-and-update/#update-wrangler) and experience no disruption to your workflow.
```
Comment (deadlypants1973) [inline content/workers/wrangler/migration/update-v2-to-v3.md]:
```suggestion
There are no special instructions for migrating from Wrangler v2 to v3. You should be able to update Wrangler by following the instructions in [Install/Update Wrangler](/workers/wrangler/install-and-update/#update-wrangler). You should experience no disruption to your workflow.
```

File: content/workers/wrangler/migration/update-v2-to-v3.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+{{<Aside type="note">}}
```
Comment (deadlypants1973) [inline content/workers/wrangler/migration/update-v2-to-v3.md]:
```suggestion
{{<Aside type="warning">}}
```

File: content/workers/wrangler/migration/update-v2-to-v3.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+{{</Aside>}}
+
+## Deprecations
+
+Refer to [Deprecations](/workers/wrangler/deprecations/#wrangler-v3) for more details on what is no longer supported in v3.
+
+## Additional assistance
+
+If you do have an issue or need further assistance, please [file an issue](https://github.com/cloudflare/workers-sdk/issues/new/choose) in the `workers-sdk` repo on GitHub.
```
Comment (deadlypants1973) [inline content/workers/wrangler/migration/update-v2-to-v3.md]:
```suggestion
If you do have an issue or need further assistance, [file an issue](https://github.com/cloudflare/workers-sdk/issues/new/choose) in the `workers-sdk` repo on GitHub.
```

File: content/workers/wrangler/migration/update-v2-to-v3.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+weight: 1
+---
+
+# Migrate from Wrangler v2 to v3
+
+There are no special instructions for migrating Wrangler v2 to v3. You should be able to update the package via [our update guide](/workers/wrangler/install-and-update/#update-wrangler) and experience no disruption to your workflow.
+
+{{<Aside type="note">}}
+
+If you tried to update to Wrangler v3 prior to v3.3 you may have experienced some compatibility issues with older operating systems. Please try again with the latest v3 where those have been resolved.
```
Comment (deadlypants1973) [inline content/workers/wrangler/migration/update-v2-to-v3.md]:
```suggestion
If you tried to update to Wrangler v3 prior to v3.3, you may have experienced some compatibility issues with older operating systems. Please try again with the latest v3 where those have been resolved.
```

**Review by deadlypants1973 — APPROVED** [review]
PCX

**Comment by mrbm** [comment]
btw just tried again, no luck. Is there an unknown, undocumented system requirement that perhaps used to work when upgrading various v2 iterations, but won't work when doing the upgrade from v2.20 to v3.x?

**Pull request merged**